pkg/arvo/sys/zuse.hoon

```diff
@@ -2155,6 +2155,114 @@
         [r s y.rp]
       ::  general recovery omitted, but possible
       --
+    ::
```

Comment:
This hoon function was [originally written](https://github.com/mrdomino/urbit/blob/jo/pedersen/pkg/arvo/sys/zuse.hoon#L2159) by @mrdomino but since it's in a branch in his fork I found it easier to just copy and paste it in here. I updated it to handle atoms bigger than 252 bits.